### ISSUE TITLE
Needs rewrite to use Forge compatible methods of applying step height attribute. Incompatible with other mods using it properly.

### ISSUE THREAD

**Issue by Xetaxheb** [spoorn/StepItUp#5]
https://nekoyue.github.io/ForgeJavaDocs-NG/javadoc/1.19.3/net/minecraftforge/common/ForgeMod.html#STEP_HEIGHT_ADDITION

Some examples: 
https://github.com/Shadows-of-Fire/Apotheosis/blob/c502c2a6c15d6a6ff9728acadccc85172f58cfe5/src/main/java/shadows/apotheosis/mixin/IForgeEntityMixin.java#L25

https://github.com/Shadows-of-Fire/Apotheosis/blob/c502c2a6c15d6a6ff9728acadccc85172f58cfe5/src/main/java/shadows/apotheosis/core/attributeslib/impl/AttributeEvents.java#L68

**Comment by spoorn** [comment]
What happens with other mods?  Looks like Apotheosis overwrites the `getStepHeight()` function, but StepItUp sets the actual variable

**Comment by gardenappl** [comment]
Can confirm that the mod does not work with Apotheosis.
```
Apotheosis-1.20.1-7.0.5.jar	   cloth-config-11.1.106-forge.jar  stepitup-2.0.1-1.20.1-forge.jar
ApothicAttributes-1.20.1-1.1.4.jar  Placebo-1.20.1-8.3.7.jar
```

**Comment by spoorn** [comment]
For future, someone in my discord reproduced this with modpacks "Medieval Minecraft 3 (forge) for mc 1.19.2 modpack via ATLauncher" and "All The Mods 8"

**Comment by victorguima** [comment]
only apotheosis needs to be active for this mod to stop working.

**Comment by SeanChengN** [comment]
I found the problem among hundreds of mods, which is incompatible with ApothicAttributes. I will continue to pay attention to the subsequent solutions.
Reference: https://github.com/Shadows-of-Fire/Apothic-Attributes/issues/7

**Comment by Xetaxheb** [comment]
> I found the problem among hundreds of mods, which is incompatible with ApothicAttributes. I will continue to pay attention to the subsequent solutions. Reference: [Shadows-of-Fire/Apothic-Attributes#7](https://github.com/Shadows-of-Fire/Apothic-Attributes/issues/7)

yes, apothic-attributes is the new name for some functions separated from apotheosis

**Comment by Nyxlanders** [comment]
Will this be fixed?

**Comment by CGMejia2001** [comment]
are there any plans to fix this?
